src/chi-vue/src/views/DataTableView.vue

```diff
@@ -81,6 +92,12 @@ import DataTableFilters from '../components/data-table-filters/DataTableFilters'
     filtersChange: e => {
       console.log(e);
     },
+    rowExpanded: e => {
+      console.log(`Expanded row: `, e);
+    },
+    rowCollapsed: e => {
+      console.log(`Collapsed row: `, e);
+    },
```

Comment:
We should remove these messages for production. If you'd like to enable logs in production, there should be a special way, but, by default, no logs should be shown in production.

Reply:
`DataTableView.vue` is not distributed in production it is a testing View.

These are the distributed components:

```
const exportComponent: ComponentLibrary = {
  components: {
    ChiSearchInput: () => import(/* webpackChunkName: "search-input" */ '../components/search-input/SearchInput'),
    ChiDataTable: () => import(/* webpackChunkName: "data-table" */ '../components/data-table/DataTable'),
    ChiDataTableToolbar: () =>
      import(/* webpackChunkName: "data-table-toolbar" */ '../components/data-table-toolbar/DataTableToolbar'),
    ChiDataTableFilters: () =>
      import(/* webpackChunkName: "data-table-filters" */ '../components/data-table-filters/DataTableFilters'),
    ChiDrawer: () => import(/* webpackChunkName: "drawer" */ '../components/drawer/drawer'),
    ChiPagination: () => import(/* webpackChunkName: "pagination" */ '../components/pagination/pagination'),
    ChiTooltip: () => import(/* webpackChunkName: "tooltip" */ '../components/tooltip/tooltip'),
  },
};
```